src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,52 @@
+import classNames from 'classnames';
+import { Link, useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../utils/getSearchWith';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const century = searchParams.getAll('century') || [];
+  const filterSex = searchParams.get('filterSex') || '';
+  const centuries = ['16', '17', '18', '19', '20'];
+
+  function handleQueryChange(event: React.ChangeEvent<HTMLInputElement>) {
+    const search = getSearchWith(
+      { query: event.target.value || null },
+      searchParams,
+    );
+
+    setSearchParams(search);
+  }
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">
+        <Link
```

Comment:
Why did you decide to delete the ready-to-use decision and implement it from scratch? ) I mean `SearchLink` component))

Don't think it is mistake but I believe it is always a good idea using ready solution

Reply:
Oh, there was a solution. I did not notice it (